migration-0.44/dashboards.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+var (
+	db *sqlx.DB
+)
+
+type Dashboard struct {
+	Id        int       `json:"id" db:"id"`
+	Uuid      string    `json:"uuid" db:"uuid"`
+	CreatedAt time.Time `json:"created_at" db:"created_at"`
+	CreateBy  *string   `json:"created_by" db:"created_by"`
+	UpdatedAt time.Time `json:"updated_at" db:"updated_at"`
+	UpdateBy  *string   `json:"updated_by" db:"updated_by"`
+	Data      string    `json:"data" db:"data"`
+	Locked    *int      `json:"isLocked" db:"locked"`
+}
+type DashboardData struct {
+	Description string              	`json:"description"`
+	Tags        []string            	`json:"tags"`
+	Name        string    				`json:"name"`
+	Layout      []Layout            	`json:"layout"`
+	Title       string              	`json:"title"`
+	Widgets     []map[string]interface{}`json:"widgets"`
```

Comment:
added the generic interfaces for `Widgets` and `Variables` since we do not need to parse them and modify anything